pyr2.html

```diff
@@ -58,31 +56,38 @@
 var a13 = qnbank[Qline][7];
 var snod = qnbank[Qline][8];
 var a14 = qnbank[Qline][9];
-document.write("<DL>");
+
+document.innerHTML="<DL>";
 document.write("<table border=0 width=550>");
-document.write("<tr>");
+document.innerHTML="<tr>";
 document.write("<td width=100% align=left>");
-document.write("<DT>", qn);
-document.write("<br><br>");
-document.write("<DD>",snoa,'. ', a11);
+document.innerHTML="<p> Burger           </p>";
+document.innerHTML="<DT>";
+document.write(qn);
```

Comment:
@cbas yes will dump document.write into the bin and replace with document.createElement and document.appendChild, will incorporate this while doing project2